src/lib/functions/runtimes/js/builders/zisi.js

```diff
@@ -34,6 +34,14 @@ const addFunctionToTree = (func, fileTree) => {
   fileTree.set(func.mainFile, { ...func, bundleFile, inputs })
 }
 
+const clearFunctionsCache = (functionsPath) => {
```

Comment:
This is required because `decache` is removing from the cache the file that was changed, but when using the zip-it-and-ship-it builder we're serving a different set of files. We need to clear the cache for those too.